src/components/ebay-notice/index.js

```diff
@@ -0,0 +1,45 @@
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+const constants = {
+    'inline': {
+        headingTag: 'span',
+        contentTag: 'span',
+        mainTag: 'div'
+    },
+    'page': {
+        headingTag: 'h2',
+        contentTag: 'div',
+        mainTag: 'section'
+    }
+};
+
+const defaults = {
+    level: 'page',
+    type: 'confirmation'
+};
+function getInitialState(input) {
```

Comment:
You should be able to remove this function entirely -- the default Marko behavior already returns `input`.

Reply:
Okay, I thought that as its in compatibility layer it might still need the getInitialState, if not i will remove it :)